src/js/services/AuthService.js

```diff
@@ -1,49 +1,69 @@
-function AuthService($http, $cookies, $mdDialog) {
+import {messages} from '../utils/constants';
+
+function AuthService($http, $q, $cookies, $mdDialog) {
     let currentUser;
     let modalIsOpen = false;
     let modal;
 
     return {
         loginToken: (token) => {
-            if (token) {
-                $http.get(`http://api.kitchen.support/account?token=${token}`)
-                .success((response) => {
-                    currentUser = response.user;
+            if (token && token.length > 0) {
+                return $http.get(`http://api.kitchen.support/account?token=${token}`)
+                .then((response) => {
+                    currentUser = response.data.user;
                 });
             }
         },
```

Comment:
What is this function for? its name makes me think it gets the login token, but it instead seems to set the current user. If this is for logging in with a token, why not move this to the `login` function?